src/types/api.d.ts

```diff
@@ -29,9 +29,13 @@ declare type PostalCode = string
 declare type Stadium = string
 declare type Stadia = Stadium[]
 declare type Distance = number
+declare type ShapValues = { [key:string]:number }
+declare type BusinessRules = { [key:string]:number }
```

Comment:
`Record<string, number>` would be cleaner imo

Reply:
Nice! didn't know about that one yet.